projects/spectator/jest/test/matchers/matchers.spec.ts

```diff
@@ -121,6 +122,16 @@ describe('Matchers', () => {
         document.querySelector('custom-element')?.shadowRoot?.querySelector("#shadow-dom")
       ).toBeHidden();
     });
+
+    it('should detect elements whose computed styles are display: none', () => {
+      window.getComputedStyle = () => ({ getPropertyValue: (style) => style == 'display' && 'none' });
+      expect(document.querySelector('#computed-style')).toBeHidden();
```

Comment:
Please check toggle functionality - hidden and than visible

Reply:
Added in [1c850d1](https://github.com/ngneat/spectator/pull/622/commits/1c850d11bfedad50e40752a6d1a22c6678398b71)